packages/components/src/ListView/Items/items.snapshot.test.js

```diff
@@ -32,4 +33,12 @@ describe('Items', () => {
 		// then
 		expect(toJson(wrapper)).toMatchSnapshot();
 	});
+
+	it('should render without toggleAll checkbox', () => {
+		// when
+		const wrapper = mount(<Items {...props} toggleAllShown={false} />);
+
+		// then
+		expect(wrapper.find('.tc-listview-toggle-all').exists()).toBeFalsy();
```

Comment:
This test is not ok.
`toggleAllShown` --> `showToggleAll`

It is weird that it passes with the props name error, I looked into the snapshots and it's not a class but an id.
`.tc-listview-toggle-all` --> `#tc-listview-toggle-all`

Since it doesn't contains only snapshot tests, can you rename the file please ?